web/src/games/chess/board.test.tsx

```diff
@@ -67,7 +52,7 @@ test('render board - all states - local friend', () => {
       currentPlayerMoves: 0,
     },
   });
-  expect(board.html()).toContain('black won');
+  expect(board.find('[data-testid="gameOverText"]')).toContainText('black_won');
```

Comment:
These strings with `_` aren't expected here. It should be `black won` because that is the translation.